src/lib/index.ts

```diff
@@ -0,0 +1,5 @@
+export { FormProvider } from "./FormProvider";
+export { FormMighty } from "./FormMighty";
+export { FormSubscriber } from "./FormSubscriber";
+export { useInitForm } from "./useInitForm";
+export type { FormState } from "./types";
```

Comment:
After my chanes this should be `export * from './types'`

Reply:
Can't
![image](https://user-images.githubusercontent.com/8074452/144722095-136fef7d-a8c8-4784-8a2b-7f7c6905f5e9.png)

Reply:
Remove the type keyword and it should work

Reply:
>> `export * from './types'`